app-office/calcurse/files/calcurse-4.6.0-tinfo.patch

```diff
@@ -0,0 +1,21 @@
+--- a/configure.ac
++++ b/configure.ac
+@@ -3,7 +3,7 @@
+ #-------------------------------------------------------------------------------
+ AC_PREREQ(2.59)
+ AC_INIT([calcurse],
+-        m4_esyscmd([build-aux/git-version-gen .version]),
++        m4_esyscmd_s([cat .version]),
+         [bugs@calcurse.org])
+ AM_INIT_AUTOMAKE
+ #m4_ifdef([AM_SILENT_RULES],[AM_SILENT_RULES([yes])])
+@@ -51,6 +51,9 @@
+   LIBS="$LIBS -l$available_ncurses"
+ ])
+ 
++AC_SEARCH_LIBS([cbreak], [tinfow tinfo],,
++    AC_MSG_ERROR([Cannot find a library providing cbreak]))
++
+ AC_CHECK_HEADERS([pthread.h], [
+     AC_CHECK_LIB(pthread, pthread_create, [
+ 	LIBS="$LIBS -pthread"
```

Comment:
You don't have to replicate the patch file if it's identical. You can call `PATCHES=( "${FILESDIR}"/name-of-the-previous.patch )`.